dataland-frontend/tests/e2e/fixtures/sfdr/SfdrDataFixtures.ts

```diff
@@ -6,12 +6,16 @@ import { generateYesNo } from "@e2e/fixtures/common/YesNoFixtures";
 import { generatePastDate } from "@e2e/fixtures/common/DateFixtures";
 
 /**
- *
- * @param undefinedProbability the probability (as number between 0 and 1) for "undefined" values in nullable fields
+ * Generates a random SFDR dataset
+ * @param setMissingValuesToNull decides whether missing values are represented by "undefined" or "null"
+ * @param missingValueProbability the probability (as number between 0 and 1) for missing values in optional fields
  * @returns SFDR object with populated properties
  */
-export function generateSfdrData(undefinedProbability = DEFAULT_PROBABILITY): SfdrData {
-  const dataGenerator = new Generator(undefinedProbability);
+export function generateSfdrData(
+  setMissingValuesToNull = false,
```

Comment:
As usual - unclear why false (i.e. undefined instead of null). Remove the option altogether and always set it to null to match what the backend would deliver.

Reply:
Done